assets/js/scenes/gameScene.js

```diff
@@ -252,6 +237,20 @@ const createGameScene = () => {
       layer("ui"),
       scale(0.4),
     ]);
+
+    // Quit Game
+    onKeyDown("escape", () => {
+      mainMusic.stop();
+      go("welcome");
+    });
+
+    // Display Quit Game Text
+    add([
+      text("Esc: Quit"),
+      pos(width() * 0.8, 0),
+      layer("ui"),
+      scale(0.4),
+    ]);
```

Comment:
Issue (non-blocking): I can't seem to see this text - is it appearing for you?

Reply:
![image](https://github.com/stephendawsondev/j-day/assets/111069103/bc95daa2-fcc5-436b-8356-007d7a87674c)